crates/bevy_gltf/src/lib.rs

```diff
@@ -9,6 +9,94 @@
 //! for loading glTF 2.0 (a standard 3D scene definition format) files in Bevy.
 //!
 //! The [glTF 2.0 specification](https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html) defines the format of the glTF files.
+//!
+//! # Quick Start
+//!
+//! Here's how to spawn a simple gltf scene
+//!
+//! ```
+//! # use bevy::prelude*;
+//! fn spawn_gltf(mut commands: Commands, asset_server: Res<AssetServer>) {
+//!     commands.spawn(SceneBundle {
+//!         // The `#Scene0` label here is very important. Keep reading for more details
```

Comment:
I'd prefer to have a bit more context about why this is needed here.

(I'd like it even better if it wasn't needed at all but...)

Reply:
How about this:

```suggestion
//!         // The `#Scene0` label here is very important it tells bevy to load the first scene in the gLTF file.
//!         // If this isn't specified bevy doesn't know which part of the gLTF file to load.
```

Reply:
Should probably be "The `#Scene0` label here is very important because it tells Bevy to load the first scene in the glTF file."